.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,24 +7,24 @@ assignees: ''
 
 ---
 
-**Describe the bug**
+### Description
 A clear and concise description of what the bug is.
 
-**To Reproduce**
+### Steps to Reproduce
 Steps to reproduce the behavior:
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
```

Comment:
That would rarely be the case for a `dart` package, more something like a called method and its parameters.